CONTRIBUTING.md

```diff
@@ -12,3 +12,7 @@ The base version represents the MAJOR and MINOR parts of [SemVer](https://semver
 
 - `main` contains the latest sources. Each merge there triggers a deploy to `nuget.org`.
 - All versions on `nuget.org` have a matching GitHub release/tag.
+
```

Comment:
Please don't update CONTRIBUTING.md as part of this pull request.
I am happy to accept a separate PR with the information.

Please note, that the requirement for Conventional Commits affects the pull request title and not the description (as the releases are automatically created based on this convention)!

Reply:
sure, I raised another PR with the updated contributing.md file here #1133